Pinta.Core/Managers/WorkspaceManager.cs

```diff
@@ -39,10 +39,35 @@ public interface IWorkspaceService
 	Document ActiveDocument { get; }
 	DocumentWorkspace ActiveWorkspace { get; }
 	event EventHandler? ActiveDocumentChanged;
+	event EventHandler<DocumentEventArgs>? DocumentCreated;
```

Comment:
Similarly, for now please only expose what's needed for effects

Reply:
This has been done, too